test/fixtures/Gemfile.rb

```diff
@@ -0,0 +1,7 @@
+gem "rake", "~> 13.0"
```

Comment:
Could this be named `Gemfile`?

Reply:
Tried before, it's not parsed by the `expectation_test_runner`: 

```ruby
TEST_FIXTURES_DIR = "test/fixtures"
TEST_FIXTURES_GLOB = File.join(TEST_FIXTURES_DIR, "**", "*.rb") # .rb expected
```

If you have any input to success it, I'm open to it 🙏